OpenRA.Mods.RA/Traits/Mine.cs

```diff
@@ -52,6 +52,9 @@ void INotifyCrushed.OnCrush(Actor self, Actor crusher, HashSet<string> crushClas
 
 		bool ICrushable.CrushableBy(Actor self, Actor crusher, HashSet<string> crushClasses)
 		{
+			if (!crusher.Info.HasTraitInfo<MineImmuneInfo>() && self.Owner.Stances[crusher.Owner] == Stance.Ally && info.AvoidFriendly)
```

Comment:
This shouldn't be needed because of https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.RA/Traits/Mine.cs#L43 ?